ownCloud/UI Elements/RoundedTableViewCell.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  RoundedTableViewCell.swift
+//  ownCloud
+//
+//  Created by Pablo Carrascal on 07/02/2019.
+//  Copyright © 2019 ownCloud GmbH. All rights reserved.
+//
+
+import UIKit
+
+class RoundedTableViewCell: UITableViewCell {
+
+	private var cornerWidth: CGFloat
+	private var cornerHeight: CGFloat
+	private var lateralInset: CGFloat
+
+	init(lateralInset: CGFloat, cornerWidth: CGFloat, cornerHeight: CGFloat) {
+		self.cornerWidth = cornerWidth
+		self.lateralInset = lateralInset
+		self.cornerHeight = cornerHeight
+		super.init(style:.default, reuseIdentifier: nil)
+	}
+
+	required init?(coder aDecoder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	override func layoutSubviews() {
+		super.layoutSubviews()
+
+		let maskLayer = CAShapeLayer()
+		let width = self.bounds.width - ((lateralInset * 2) + (safeAreaInsets.left * 2))
```

Comment:
safeAreaInsets.left can be differ from safeAreaInsets.right
I think you should add the safeAreaInsets.right instead of multiply

Reply:
You're right, I changed it.